src/Java.Interop.Tools.JavaCallableWrappers/Java.Interop.Tools.JavaCallableWrappers/Crc64.cs

```diff
@@ -170,7 +173,8 @@ public class Crc64 : HashAlgorithm
 		    0x536fa08fdfd90e51, 0x29b7d047efec8728,
 		};
 
-		ulong crc;
+		ulong crc = ulong.MaxValue;
+		ulong length;
```

Comment:
I'd feel better if `length` was initialized to 0. It isn't set to anything anywhere in the code and even though it is initialized now to the default value of the type, it may change some day.